code/game/objects/effects/decals/posters/bs12.dm

```diff
@@ -303,3 +303,74 @@
+	icon_state="bsposter64"
+	name = "Rob Boss is Watching You!"
+	desc = "You see the famous artist Rob Boss. His shows about drawing still lifes, various creatures and phenomena are actively promoted by NanoTrasen. At the top of the poster you can see the inscription - Rob Boss is Watching You."
+
+/datum/poster/bay_65
+	icon_state="bsposter65"
+	name = "ETA 00:00"
+	desc = "This poster captures the whole point of the crew being late to the departure shuttle. Nuff said."
+
+/datum/poster/bay_66
+	icon_state="bsposter66"
+	name = "the call of Zen"
+	desc = "It is a poster of the Yin-Yang symbol, often associated with the now unpopular religion of Taoism.It's a very calming image."
+
+/datum/poster/bay_67
+	icon_state="bsposter67"
+	name = "bioshocked"
+	desc = "This poster depicts a huge, strange steel spacesuit; perhaps it's needed to find some gel-type race in space.At the top you can see Bioshocked, which must have hit that gelatinous race pretty hard."
+
+/datum/poster/bay_68
+	icon_state="bsposter68"
+	name = "crazy Dude"
+	desc = "The poster depicts a red-haired man wearing glasses. He has a crazy grin on his face. We can only hope that the blood on the poster is not real.It's not real, is it?"
+
+/datum/poster/bay_69
+	icon_state="bsposter69"
+	name = "don't run with scissors!"
+	desc = "The poster shows a road sign with scissors drawn on it. Perhaps you really shouldn't be running down the halls with scissors. That's good advice."
+
+/datum/poster/bay_70
+	icon_state="bsposter70"
+	name = "pipe of love!"
+	desc = "The poster depicts the health pipe, an old, long-forbidden and long-forgotten drug that is instantly addictive and makes you feel perfectly healthy."
+
+/datum/poster/bay_71
+	icon_state="bsposter71"
+	name = "cheers everyone!"
+	desc = "This poster greets you with a scruffy-looking young man wearing some kind of space is cool! t-shirt. He's holding a glass full of white liquid, and he looks like he wants to celebrate something-just as long as there's a reason."
+
+/datum/poster/bay_72
+	icon_state="bsposter72"
+	name = "metroidvania"
+	desc = "The poster has a picture of a metroid on it. At the bottom, in small print, you can read. This poster is designed and implemented by the Universal Metroid Support Association. Metroid to the masses! Say No! to the genocide of Jell-O Friends!"
+
+/datum/poster/bay_73
+	icon_state="bsposter73"
+	name = "just drive"
+	desc = "This poster shows an unsightly jacket from the distant past. The same jackets worn by those who like to rummage through old clothes from centuries ago. The overall style of the poster tries to show its old-fashionedness with all its appearance. What the written inscription Drive calls for is unclear."
+
+/datum/poster/bay_74
+	icon_state="bsposter74"
+	name = "make coffee, not sabotage!"
+	desc = "Another NanoTrasen propaganda aimed at employees who are not the most loyal.The call to drink coffee instead of sabotage should have worked well, they thought. Judge yourself."
```

Comment:
```suggestion
	desc = "Another NanoTrasen propaganda aimed at employees who are not the most loyal. The call to drink coffee instead of sabotage should have worked well, they thought. Judge yourself."
```